.gitignore

```diff
@@ -1,31 +1,40 @@
+# Ignore Python compiled files
 *.pyc
-.idea/
-*~
-*.swp
+
+# Ignore IDE-specific files and directories
+.idea/        # JetBrains IDE settings
```

Comment:
Thanks for this change @Nadav23AnT - one minor thing, could you align the comment here with the other IDEs?

Reply:
Pushed change to fix.

Reply:
Thank you @loadams.